code/mob/living/silicon.dm

```diff
@@ -61,6 +61,10 @@
 
 /mob/living/silicon/proc/update_canmove()
 	canmove = !(src.hasStatus(list("weakened", "paralysis", "stunned")) || buckled)
+	if (!canmove)
```

Comment:
i'll clean up this mess after the buckling stuff is in, so as to break less stuff